packages/libsql-client/src/sqlite3.ts

```diff
@@ -167,9 +170,7 @@ export class Sqlite3Client implements Client {
         }
     }
 
-    async migrate(
-        stmts: Array<InStatement>,
-    ): Promise<Array<ResultSet>> {
+    async migrate(stmts: Array<InStatement>): Promise<Array<ResultSet>> {
```

Comment:
Diff noise.

Reply:
Prettier ran on commit, have the rules changed?